packages/rabbitmq-server/build.sh

```diff
@@ -0,0 +1,37 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/rabbitmq/rabbitmq-server
+TERMUX_PKG_DESCRIPTION="Feature rich, multi-protocol messaging and streaming broker"
+TERMUX_PKG_LICENSE="MPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="4.0.3"
+TERMUX_PKG_SRCURL=https://github.com/rabbitmq/rabbitmq-server/releases/download/v${TERMUX_PKG_VERSION}/rabbitmq-server-generic-unix-${TERMUX_PKG_VERSION}.tar.xz
+TERMUX_PKG_SHA256=607ba3b4fd1fc133f3bada1b2225ff0527f99cc6eb32423ae73f6c9f5ee9bd13
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_HOSTBUILD=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS="erlang"
+TERMUX_PKG_PLATFORM_INDEPENDENT=true
```

Comment:
There's a couple things that should be removed as they are not applicable for termux.
- We don't have support for VMWare, so `vmware-rabbitmq` is not usable.
<sup>(Unless it does something other than what I'd expect based on the name, but there is no man page to check)</sup>
- Unpatched versions of RabbitMQ would not work on Termux, so `rabbitmq-upgrade` is not usable.
- `rabbitmq-service` is for running RabbitMQ as a Windows service, so we shouldn't ship the man page for it.
- Same for `rabbitmq-echopid`, also Windows specific.
```suggestion
TERMUX_PKG_PLATFORM_INDEPENDENT=true
TERMUX_PKG_RM_AFTER_INSTALL="
bin/rabbitmq-upgrade
bin/vmware-rabbitmq
lib/rabbitmq/lib/rabbitmq_server-${TERMUX_PKG_VERSION}/escript/rabbitmq-upgrade
lib/rabbitmq/lib/rabbitmq_server-${TERMUX_PKG_VERSION}/escript/vmware-rabbitmq
share/man/man8/rabbitmq-upgrade.8.gz
share/man/man8/rabbitmq-service.8.gz
share/man/man8/rabbitmq-echopid.8.gz
"
```